showcase/index.js

```diff
@@ -107,6 +109,8 @@ export const showCase = {
   SimpleTreemap,
   TreemapExample,
 
+  BasicSunburst,
```

Comment:
right now they are broken apart by group, but that is definately something to do in the future, specifically i think itll be addressed in #343